meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher.service

```diff
@@ -2,11 +2,11 @@
 Description=Resin init flasher service
 Requires=\
     mnt-boot.mount \
```

Comment:
I can, but I usually saw in our code that we go for all the dependency explicitly in all services, we do not take into account that some dependent service depends on one's dependencies. I reckoned we do this for clarity in our code.